### DISCUSSION TITLE
“网址”没有备注（可选）

### DISCUSSION THREAD

**Discussion by Pathsis** [walinejs/waline#662] [Q&A] [answered]
是否可以改进下下呢呀~

**Comment by Mister-Hope** [comment, marked as answer]
网址在默认配置下本来也不是可选的

**Reply by Pathsis** [reply]
感谢回复！现在是有点对用户不太友好的：
1 . 用户可能因为看到“登录”按钮而以为需要注册和登录才能评论；
2. 用户不知道不填任何信息也可以评论；
3. 用户不知道是否需要三种信息都填上才能评论。
我的建议是提供一些基本提示符：
1 . 网站（可选）
2. 评论框占位符：您无需登录或填写任何信息而可以直接匿名评论。现在的“欢迎评论”没有提供任何有用的信息。

**Reply by Mister-Hope** [reply]
你的建议都可以自己配置。建议回头自己看文档。

**Reply by Mister-Hope** [reply]
另外你说的1，当你把对应的内容设置成非必填之后，我认为会出现”可选“的提示。

**Reply by Pathsis** [reply]
> 另外你说的1，当你把对应的内容设置成非必填之后，我认为会出现”可选“的提示。

感谢回复。不好意思，我小白一个，刚用上。目前只是觉得有些令人困惑。

**Reply by Mister-Hope** [reply]
Waline 本身就支持”仅允许注册用户评论“ ”不开启注册登陆功能“ 和 “两者并存” 为三种情况分别设置一个特别长的评论框占位符是 **没有任何道理的** 。 如果你自己有相关配置需求，并期望给你用户更明确的指引，评论框占位符你是可以自己配置。

**Reply by Pathsis** [reply]
> Waline 本身就支持”仅允许注册用户评论“ ”不开启注册登陆功能“ 和 “两者并存” 为三种情况分别设置一个特别长的评论框占位符是 **没有任何道理的** 。 如果你自己有相关配置需求，并期望给你用户更明确的指引，评论框占位符你是可以自己配置。

感谢指教！请问我如何配置占位符？

**Reply by Mister-Hope** [reply]
https://waline.js.org/guide/client/i18n.html

**Reply by Pathsis** [reply]
> https://waline.js.org/guide/client/i18n.html

谢谢！但我不知道如何配置这“local”。

**Reply by Pathsis** [reply]
> > https://waline.js.org/guide/client/i18n.html
> 
> 谢谢！但我不知道如何配置这“local”。

请问locale在哪里配置啊~不胜感激

**Reply by Pathsis** [reply]
> Waline 本身就支持”仅允许注册用户评论“ ”不开启注册登陆功能“ 和 “两者并存” 为三种情况分别设置一个特别长的评论框占位符是 **没有任何道理的** 。 如果你自己有相关配置需求，并期望给你用户更明确的指引，评论框占位符你是可以自己配置。

不会配置locale，只能弱弱地再次建议：按照官方的指引默认情况下，评论框提示符，“**您无需登录或填写任何信息而可直接匿名评论**。”，应该不是一个很长的占位符。

**Reply by Mister-Hope** [reply]
```js
Waline({
  // ...
  locale : {
    placeholder: 'xxxxxxx'
  }
});
```

**Reply by Pathsis** [reply]
感谢！但我不知道到哪里填写这代码。算啦，我就这样使用啦😁
